### PR TITLE
Add GLUE fine-tuning guide for KerasHub

### LINKED ISSUE #2401: Add MobileNetV5 to KerasHub
MobileNetV5 is a vision encoder used within Google's multimodal Gemma 3n family of models. It's a lightweight convolutional neural network specifically designed for on-device or embedded applications, prioritizing high efficiency, low latency, and low power usage.
The MobileNet series is known for its use of depthwise separable convolutions to significantly reduce the number of parameters and computational cost compared to traditional convolutional neural networks. This makes them ideal for tasks like image classification, object detection, and semantic segmentation on devices with limited resources, such as mobile phones.

Link to weights: https://huggingface.co/timm/mobilenetv5_300m.gemma3n

### PR DESCRIPTION
**Pull request by bitanb1999**
## Summary
- Adds a keras.io guide (`guides/keras_hub/finetuning_on_glue_with_keras_hub.py`) that fine-tunes `keras_hub.models.BertTextClassifier` on GLUE/MRPC end-to-end: loading the dataset via `tensorflow_datasets`, fine-tuning `bert_tiny_en_uncased`, evaluating, and predicting on new sentence pairs.
- Links out to the existing [`benchmarks/glue.py`](https://github.com/keras-team/keras-hub/blob/master/benchmarks/glue.py) script in keras-hub for a configurable CLI sweep over models/presets.
- Registers the guide in `HUB_GUIDES_MASTER` (`scripts/hub_master.py`).
- `.md`/`.ipynb` generated via `python autogen.py add_guide keras_hub/finetuning_on_glue_with_keras_hub` (actually executed, real outputs included).

Fixes keras-team/keras-hub#2401 (follow-up to keras-team/keras-io#2403).

## Test plan
- [x] Ran `black` on the guide source per tutobook conventions
- [x] Ran `python autogen.py add_guide keras_hub/finetuning_on_glue_with_keras_hub` end-to-end (downloads BERT-tiny + GLUE/MRPC, fine-tunes 3 epochs, evaluates, predicts) with no errors
- [x] Verified `predict()` output are genuine class probabilities (softmax) that sum to 1 and make sense for the example sentence pairs

### CODE REVIEW

**Comment by bitanb1999** [comment]
@maitry63 @laxmareddyp could you please approve the workflows? I have already rebased the branch to upsteam/master to ensure the docker image is successfully built and the workflow goes through.

**Comment by laxmareddyp** [comment]
Hi @bitanb1999 

Thanks for adding this guide, and apologies for the delay in reviewing! I will take a look at it this week. In the meantime, could you please resolve the merge conflicts? Thanks!"

File: guides/keras_hub/finetuning_on_glue_with_keras_hub.py [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+    num_classes=2,
+    activation="softmax",
+)
+
+"""
+Because the classifier carries its own preprocessor, we can pass raw text
+straight to `fit()` -- there's no need for a separate tokenization step.
+"""
+
+train_ds = train_ds.batch(BATCH_SIZE).prefetch(tf.data.AUTOTUNE)
```
Comment (laxmareddyp) [inline guides/keras_hub/finetuning_on_glue_with_keras_hub.py:107]: Nit: train_ds is being shadowed here — it was already assigned at line 63 (from tfds.load), then reassigned at line 75 (after .map), and now reassigned again after .batch(). In a notebook/Colab context, if a user re-runs this cell without re-running the previous cells, train_ds is already batched and batching it again will silently change the batch structure. Consider using a distinct variable name for the batched version
```
train_batches = train_ds.batch(BATCH_SIZE).prefetch(tf.data.AUTOTUNE)
validation_batches = validation_ds.batch(BATCH_SIZE).prefetch(tf.data.AUTOTUNE)
```

Then use train_batches / validation_batches in fit(), evaluate(), etc. below.

**Review by laxmareddyp — CHANGES_REQUESTED** [review]
Left one nit comment , The final val accuracy here is ~72.5%, which is barely above the MRPC majority-class baseline (~68%). Could you add a sentence setting expectations — something like:

With bert_tiny_en_uncased (4.4M params), expect validation accuracy around 70-75%. bert_base_en_uncased typically reaches more on MRPC.

This helps readers understand the low accuracy is expected for the tiny model and not a problem with the workflow.

Pease generate ipynb and md files  once you address the comment. 
Thanks
